packages/twenty-website/src/content/developers/self-hosting/upgrade-guide.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade guide
+icon: TbCloud
+image: /images/user-guide/notes/notes_header.png
+---
+
+## General guidelines 
+
+Always make sure to backup your database before starting the upgrade process.
+
+## If you used Docker Compose
+
+1. In a terminal, on the host where Twenty is running, turn off Twenty: `docker compose down`
+
+2. Upgrade the version by changing the `TAG` value in the .env file near your docker-compose.
+
+4. Bring Twenty back online with `docker-compose up -d`
+ 
+## Data Migration
+
+Migrating a CRM is a bit trickier than migrating a traditional software, because the data model of Standard Object has to evolve smoothly along your custom objects.
+
+## v0.21.0 to v0.22.0
+
+Run the following commands:
+
+```
+yarn database:migrate:prod 
+yarn command:prod workspace:sync-metadata -f 
+yarn command:prod upgrade-0.22
```

Comment:
Is it ok to run it after sync:metadata? I assume so but just checking!

Reply:
Yes it's even mandatory here for things to work!, otherwise one of the commands (to add the new address field introduced by the sync-metadata to the views) won't work and will require you to run sync-metadata first and re-run after